README.md

```diff
@@ -154,9 +154,9 @@ ensure that the most popular native Node addons compile from source. Those are:
 
 ## License & Credits
 
-The Python installation was made possible by [Ali Hajimirza](https://github.com/ali92hm), who kindly wrestled with Python's MSIs until they surrendered. For details regarding the license agreements applicable to Python, see *History and License* [2.x](https://docs.python.org/2/license.html) and [3.x](https://docs.python.org/3/license.html).
+The Python installation was made possible by [Ali Hajimirza](https://github.com/ali92hm), who kindly wrestled with Python's MSIs until they surrendered. For details regarding the license agreements applicable to Python, see *History and License* [3.x](https://docs.python.org/3/license.html).
 
 Use of Microsoft software is subject to the terms of the corresponding license agreements. For details regarding the license agreements applicable to Visual Studio products, refer to their [*License Directory* page](https://visualstudio.microsoft.com/license-terms/). (See also [this discussion](https://social.msdn.microsoft.com/Forums/en-US/08d62115-0b51-484f-afda-229989be9263/license-for-visual-c-2017-build-tools?forum=visualstudiogeneral) for the gist of it.)
 
-Copyright (C) 2018 Felix Rieseberg. Licensed MIT. For more details, please see LICENSE.
+Copyright (C) 2020 Felix Rieseberg. Licensed MIT. For more details, please see LICENSE.
```

Comment:
I have seen that people bump the copyright year but have never understood what value this adds. Do you have any links you could share to help me understand? So far I found these [SO](https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated) [questions](https://softwareengineering.stackexchange.com/questions/130478/what-copyright-date-for-an-update-to-an-open-source-project-from-last-year) and [this legal article](https://www.nolo.com/legal-encyclopedia/when-do-you-need-copyright-notice-software-and-where-do-you-place-it.html) but it still isn't clear to me what the best practice is and what the purpose is.

Reply:
I have read enough to understand that this should be reverted.

Reply:
> I have read enough to understand that this should be reverted.

That's fine, though I still wish it made more sense to me. On one hand we are talking about "first publication" (so should use older) but on the other we are talking about a new version of the software (so should use newer). 

When I run `cmd.exe` I see the following notice displayed:
```
Microsoft Windows [Version 10.0.18362.535]
(c) 2019 Microsoft Corporation. All rights reserved.
```

Clearly the Microsoft Windows product family and even Windows 10 have been around for far longer than 1 year... 🤷‍♂

Reply:
I didn't mean to [bike-shed](https://exceptionnotfound.net/bikeshedding-the-daily-software-anti-pattern/) with this, BTW. Just didn't think it should be mixed in with the rest of the (unrelated) PR.

Reply:
My sense was _if new copyrightable changes are made then the date should be updated_ and I do not think that these mods reach that threshold.